src/pkg/services/m365/api/drive_pager_test.go

```diff
@@ -0,0 +1,65 @@
+package api_test
+
+import (
+	"testing"
+
+	"github.com/alcionai/clues"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/alcionai/corso/src/internal/tester"
+)
+
+type DrivePagerIntgSuite struct {
+	tester.Suite
+	its intgTesterSetup
+}
+
+func TestDrivePagerIntgSuite(t *testing.T) {
+	suite.Run(t, &MailPagerIntgSuite{
+		Suite: tester.NewIntegrationSuite(
+			t,
+			[][]string{tester.M365AcctCredEnvs}),
+	})
+}
+
+func (suite *DrivePagerIntgSuite) SetupSuite() {
+	suite.its = newIntegrationTesterSetup(suite.T())
+}
+
+func (suite *DrivePagerIntgSuite) TestGetItemsInContainerByCollisionKey() {
```

Comment:
this should probably be checking for known items since we already know what the collision key will be. You can make a folder with specific items to test with if needed or use something like gock

Reply:
yup, good call.